drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -89,19 +89,24 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
     age_oldest_to_youngest: 'Age: Oldest to Youngest',
   }.freeze
 
-  scope :with_access, ->(user, *permissions) do
+  # Enrollments at Projects where the user has the specified permission(s).
+  # WARNING UNSAFE! This does not check for can_view_project or can_view_enrollment_details.
+  # This scope should almost always be used in conjunction with viewable_by.
```

Comment:
We don't use this scope anywhere except in the `viewable_by` scope (just below).

We might want to use it elsewhere in the future, though. For example, filtering down a list of a client's enrollments to only the ones you can edit `viewable_by(user).with_access(user, :can_edit_enrollment_details)`. We don't do that anywhere, yet, but we may in the future. So, decided to leave this scope available with a warning.

Reply:
And yeah, this scope was _already_ unsafe to use on its own, since it doesn't check for "can_view_enrollment_details".